pom.xml

```diff
@@ -162,6 +162,15 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
+            <modules>
+                <module>symphony-bdk-bom</module>
+                <module>symphony-bdk-core</module>
+                <module>symphony-bdk-core-invokers</module>
+                <module>symphony-bdk-examples</module>
```

Comment:
this one should be excluded out of the release profile, shouldn't it?

Reply:
Bad copy-paste 😖